src/Android/Xamarin.Android/Samples/Layers/ChangeStretchRenderer/ChangeStretchRenderer.cs

```diff
@@ -0,0 +1,345 @@
+
+            // Add the sub layout to the main layout
+            layout.AddView(subLayout2);
+
+            // ------------
+
+            // Create button to change stretch renderer of the raster
+            _UpdateRenderer = new Button(this);
+            _UpdateRenderer.Text = "UpdateRenderer";
+            // Hook to touch/click event of the button
+            _UpdateRenderer.Click += OnUpdateRendererClicked;
+            layout.AddView(_UpdateRenderer);
+
+            // Create the map view and add it to the main layout
+            _myMapView = new MapView(this);
+            layout.AddView(_myMapView);
+
+            // Set the layout as the sample view
+            SetContentView(layout);
+        }
+
+        private async void Initialize()
+        {
+            // Create a simple string array of the human-readable stretch renderer names from the ObservableCollection
+            string[] myStringArray_RendererTypes = { "Min Max", "Percent Clip", "Standard Deviation" };
+
+            // Create an ArrayAdapter from the simple string array
+            ArrayAdapter myArrayAdapter_LayerNamesInTheMap = new ArrayAdapter<string>(this, Android.Resource.Layout.SimpleListItem1, myStringArray_RendererTypes);
+
+            // Display the human-readable stretch renderer names to be displayed a ListView
+            _RendererTypes.Adapter = myArrayAdapter_LayerNamesInTheMap;
+
+            // Add an imagery basemap
+            Map myMap = new Map(Basemap.CreateImagery());
+
+            // Wait for the map to load
+            await myMap.LoadAsync();
+
+            // Get the file name
+            string filepath = await GetRasterPath();
+
+            // Load the raster file
+            Raster myRasterFile = new Raster(filepath);
+
+            // Create the layer
+            RasterLayer myRasterLayer = new RasterLayer(myRasterFile);
+
+            // Add the layer to the map
+            myMap.OperationalLayers.Add(myRasterLayer);
+
+            // Wait for the layer to load
+            await myRasterLayer.LoadAsync();
+
+            // Set the viewpoint
+            myMap.InitialViewpoint = new Viewpoint(myRasterLayer.FullExtent);
+
+            // Add map to the mapview
+            _myMapView.Map = myMap;
+        }
+
+        private void OnUpdateRendererClicked(object sender, EventArgs e)
+        {
+            // This function acquires the user selection of the stretch renderer from the table view
+            // along with the parameters specified, then a stretch renderer is created and applied to 
+            // the raster layer
+
+            // Get the user choice for the raster stretch render
+            string myRendererTypeChoice;
+
+            if (_RendererTypes.SelectedItem == null)
+            {
+                // If the user does not click on a choice in the table but just clicks the
+                // button, the selected value will be null and we will use the initial
```

Comment:
Although I personally like the 'we' language (and am probably guilty of using it in many of my own samples), I'm told this is inconsistent with the Esri technical writing style. This is such a minor nitpick that I feel bad mentioning it, but I think maybe we'll want to think about it in the future.